test/test/async_check_cancel.cpp

```diff
@@ -111,8 +187,10 @@ TEST_CASE("async_check_cancel", "[.clang-crash]")
 TEST_CASE("async_check_cancel")
 #endif
 {
-    Check(Action);
-    Check(ActionWithProgress);
-    Check(Operation);
-    Check(OperationWithProgress);
+    //Check(Action);
```

Comment:
what should happen to these commented out checks?

Reply:
whoops will restore. commented out for easier debugging